### PR TITLE
Add a minor clarification

### PR DESCRIPTION
**Pull request by mrexodia**
### What does this PR try to resolve?

It tries to clarify the overriding build scripts section. We spent over an hour with 3 people trying to get it to work and it was extremely difficult to figure out.

### How should we test and review this PR?

N/A

### Additional information

N/A

### CODE REVIEW

File: src/doc/src/reference/build-scripts.md [hunk 1]
```diff
@@ -444,7 +444,7 @@ prevent running the build script in question altogether and instead supply the
 metadata ahead of time.
 
 To override a build script, place the following configuration in any acceptable
-Cargo [configuration location](config.md).
+Cargo [configuration location](config.md) (e.g. `config.toml`, _not_ `Cargo.toml`).
```
Comment (epage) [inline src/doc/src/reference/build-scripts.md:447]: While I understand you were confused on this point, I'm not too sure the value of calling this out specifically.

Ignoring the consistency that cargo has in referring to config vs manifests, the link fairly quickly resolves ambiguity.

In contrast, this has the potential to add noise to the documentation which can have the opposite effect of clarifying, particularly if the question is asked "where else could someone get confused on these points?" and we proceed to annotate those.  If we say its only this place, what is different about this location compared to others?

As an alternative, we could possibly just do `in [any relevant `config.toml` file](...)` as it looks like we've shifted the focus of the documentation away from the other accepted file names to this one.

Comment (epage) [reply src/doc/src/reference/build-scripts.md:447]: (granted, that alternative carries the risk of people who still use the other file name to read it and think it doesn't apply to them)

Comment (mrexodia) [reply src/doc/src/reference/build-scripts.md]: Thanks for the feedback. I updated it to instead link to `config.toml`.

**Pull request merged**